modules/ROOT/pages/7.1.2-release-notes.adoc

```diff
@@ -55,17 +55,21 @@ For information on the **<Open source plugin name>** plugin, see xref:<plugincod
 
 The following premium plugin updates were released alongside {productname} {release-version}.
 
-=== <Premium plugin name 1> <Premium plugin name 1 version>
+=== Spell Checker
+// TINY-10904
 
-The {productname} {release-version} release includes an accompanying release of the **<Premium plugin name 1>** premium plugin.
+The {productname} {release-version} release includes an accompanying release of the **Spell Checker** premium plugin.
 
-**<Premium plugin name 1>** <Premium plugin name 1 version> includes the following <fixes, changes, improvements>.
+**Spell Checker** includes the following fix.
```

Comment:
```suggestion
**Spell Checker** premium plugin includes the following fix.
```
Should be consistent with the other entry, https://github.com/tinymce/tinymce-docs/pull/3311

Reply:
removing this entry as its not required.